apps/emqx_management/src/emqx_mgmt_api_plugins.erl

```diff
@@ -563,6 +685,18 @@ aggregate_status([{Node, Plugins} | List], Acc) ->
         ),
     aggregate_status(List, NewAcc).
 
+format_plugin_schema_with_i18n(NameVsn) ->
+    #{
+        avsc => try_read_file(fun() -> emqx_plugins:plugin_avsc(NameVsn) end),
+        i18n => try_read_file(fun() -> emqx_plugins:plugin_i18n(NameVsn) end)
+    }.
+
+try_read_file(Fun) ->
+    case Fun() of
+        {ok, Bin} -> Bin;
```

Comment:
not a binary
`emqx_plugins:plugin_avsc/1` and `emqx_plugins:plugin_i18n/1` will return json object (a map).